packages/server/src/fcm/fcm.service.ts

```diff
@@ -0,0 +1,51 @@
+import { Injectable } from "@nestjs/common";
+import * as FCM from "fcm-node";
+import { Data } from "./fcm.interfaces";
+
+@Injectable()
+export class FcmService {
+  private readonly serverKey;
+  private readonly fcm;
+  constructor() {
+    // this.serverKey = this.configService.get("serverKey");
+    // this.fcm = new FCM(this.serverKey);
+  }
+
+  private message(to: string, data: Data, collapse_key: string) {
+    return {
+      to,
+      collapse_key,
+      notification: {
+        title: data.title,
+        body: data.body,
+        click_action: "Result3Activity",
+      },
+      data,
+    };
+  }
```

Comment:
title, body가 not nullable한 속성이라면, 위에 검사하는 로직을 넣는게 좋을거같습니다.

```suggestion
  private message(to: string, data: Data, collapse_key: string) {
    const {title, body} = data;
    
    if(_.isNil(title) || _.isNil(body)){
        // some logics...
    }
    return {
      to,
      collapse_key,
      notification: {
        title: data.title,
        body: data.body,
        click_action: "Result3Activity",
      },
      data,
    };
  }
```

Reply:
수정했습니다.